web/forms.py

```
@@ -92,6 +93,7 @@
     "GradeableLinkForm",
     "LinkGradeForm",
     "AwardAchievementForm",
+    "WaitingRoomForm",
```

Ruff (0.8.2)

Undefined name `WaitingRoomForm` in `__all__`

(F822)